accesshandler/pkg/providers/aws/flask/validate.go

```diff
@@ -0,0 +1,63 @@
+package flask
+
+import (
+	"context"
+	"errors"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/identitystore"
+	"github.com/aws/aws-sdk-go-v2/service/organizations"
+	orgtypes "github.com/aws/aws-sdk-go-v2/service/organizations/types"
+	"github.com/common-fate/granted-approvals/accesshandler/pkg/diagnostics"
+	"github.com/common-fate/granted-approvals/accesshandler/pkg/providers"
+)
+
+// Validate the access against AWS SSO without actually granting it.
+// This provider requires that the user name matches the user's email address.
+
+func (p *Provider) ensureAccountExists(ctx context.Context, accountID string) error {
+	_, err := p.orgClient.DescribeAccount(ctx, &organizations.DescribeAccountInput{
+		AccountId: &accountID,
+	})
+	var anf *orgtypes.AccountNotFoundException
+	if errors.As(err, &anf) {
+		return &AccountNotFoundError{AccountID: accountID}
+	}
+
+	return err
+}
+func (p *Provider) ValidateConfig() map[string]providers.ConfigValidationStep {
```

Comment:
Can you please add some extra config validations?

- try to assume both the cluster access role as well as the SSO access role (should be two separate checks)
- try to describe tasks in the cluster